fastestimator/network.py

```diff
@@ -274,7 +275,20 @@ def load_epoch(self, mode: str, epoch: int, output_keys: Optional[Set[str]] = No
         super().load_epoch(mode, epoch, output_keys, warmup)
         if self.device.type == "cuda":
             for model in self.epoch_models:
+                # move model variables to gpu
                 model.to(self.device)
+                # move optimizer variables to gpu
+                self._move_between_device(model.current_optimizer.state, self.device)
+
+    def _move_between_device(self, var, device):
```

Comment:
is the idea here that we don't want to leave scheduled optimizers on the GPU?

Reply:
yes.